### DISCUSSION TITLE
Trunking Recorder?

### DISCUSSION THREAD

**Discussion by The-Elevator-Guy** [chuot/rdio-scanner#148] [Q&A]
I'm a casual user of Rdio that wanted to try his hand at setting up my own system. My county is a Motorola smart net II analog system. Ive been able to set up some rtl-sdr's, and using unitrunker, virtual audio cable and TRUNKING recorder, I've been able to get to the point that I can use trunking recorders web panel to play calls. So, here's my question. I had originally though that the program I was using, Trunking recorder, was compatible with Rdio.  I see now that I actually need TRUNK recorder, and that there is no windows application for this. Anything I've done with my scanners has always been windows.

So, is there a windows program that is compatible with Rdio that I can use along with Unitrunker? Any help is greatly appreciated.

**Comment by chuot** [comment]
[sdrtrunk](https://github.com/DSheirer/sdrtrunk) is one of them. There are more, but I'd rather let the Windows folks answer this one since I'm more of a Unix/Linux guy. Basically, as long as the software can generate audio files split by conversation or transmission, Rdio Scanner will pick them up with a dirWatch.

**Reply by The-Elevator-Guy** [reply]
Thank you for the reply. Is there a particular format that the program should follow when naming the files ? It can definitely save each individual transmission. It does put them in different folders based on the date and talk group. Is that ok for rdio, or should it just be dumping all the audio into 1 folder?

**Reply by The-Elevator-Guy** [reply]
And, one more question if you don't mind. I 'think' I have rdio set to look at the correct directory for audio files, but it keeps saying "dirwatch.ingest: no system" Is it because I don't have the system ID set properly? It's asking for a decimal format, and I only have it in Hex, which is 4F3B.

**Comment by chuot** [comment]
Dirwatch can search folders recussively, so no problem having your files in subfolders organized by days. However, be sure to `deleteAfter` ingestion as looking at too many files can slow down Rdio Scanner considerably. The #SYS meta tag will only read numbers (dec), it won't work if your files have the system id in hex in their filename. If possible, organize your files so that each system is in its own folder, then force the system id into the dirwatch definition. A suggested filename pattern would be something like this: `sysname_#TG_#DATE_#TIME_#HZ`. Also be sure to have defined the correct file extension in the dirwatch definition.
